setup.py

```diff
@@ -66,12 +61,15 @@ def is_raspberry_pi(raise_on_errors=False):
     description="A library which communicates with ZiGate radios for zigpy",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="http://github.com/doudz/zigpy-zigate",
+    url="http://github.com/zigpy/zigpy-zigate",
     author="Sébastien RAMAGE",
     author_email="sebatien.ramage@gmail.com",
     license="GPL-3.0",
     packages=find_packages(exclude=['*.tests']),
-    install_requires=requires,
```

Comment:
This is needed because during install, if we are on Raspberry Pi board then we add a dependency to Rpi.GPIO